demo/index.html

```diff
@@ -34,11 +35,16 @@
     margin: 0;
     padding: 0;
   }
+  url-bar {
+    background-color: white;
```

Comment:
In many cases, it is rather difficult to predict the the most correct color palette automatically. It seems like the best outcome is to minimize surprising behavior and maximize user control of the outcome.

Regarding surprising behavior:

- My URL bar does not change color on desktop (barring a fancy browser theme).
- My URL bar sometimes changes color on mobile, but not because it is transparent. In Chrome, this can be configured with [`meta theme-color`](https://developers.google.com/web/fundamentals/design-and-ui/browser-customization/theme-color?hl=en).

Regarding maximizing user control: While I can style this element when I use it directly, I need a custom property or mixin to customize it across style scopes.